BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewLoginItem/ViewLoginItemView.swift

```diff
@@ -32,10 +32,12 @@ struct ViewLoginItemView: View {
             passkeyRow(fido2Credential)
         }
 
-        if !store.state.isTOTPAvailable {
-            premiumSubscriptionRequired
-        } else if let totpModel = store.state.totpCode {
-            totpRow(totpModel)
+        if let totpModel = store.state.totpCode {
```

Comment:
🤔 super small nit, andI know its technically the same logic here but would it make more sense to swap the conditions here for readability?  feel free to ignore!

```
if store.state.isTOTPAvailable {
    if let totpModel = store.state.totpCode {
        totpRow(totpModel)
    }
} else {
    premiumSubscriptionRequired
}
```